internal/twirptest/importer_local/importer_local.proto

```diff
@@ -0,0 +1,10 @@
+syntax = "proto3";
+
+package twirp.internal.twirptest.importer_local;
+option go_package = "github.com/twitchtv/twirp/internal/twirptest/importer_local";
+
+import "importer_local_msgdef.proto";
+
+service Svc {
```

Comment:
Could you add a comment (wherever you deem appropriate) on what this package is testing for future reference?